node/dataSources/session.ts

```diff
@@ -0,0 +1,46 @@
+import { RequestOptions, RESTDataSource } from 'apollo-datasource-rest'
+import { ColossusContext } from 'colossus'
+import { forEachObjIndexed } from 'ramda'
+
+export interface SegmentData {
+  campaigns?: any
+  channel: string
+  priceTables?: any
+  regionId?: string
+  utm_campaign?: string
+  utm_source?: string
+  utmi_campaign?: string
+  currencyCode: string
+  currencySymbol: string
+  countryCode: string
+  cultureInfo: string
+}
+
+export class SessionDataSource extends RESTDataSource<ColossusContext> {
+  constructor() {
```

Comment:
This constructor implementation is really required?

Reply:
I don't think so. I was following the pattern from other data sources.